tools/build_defs/repo/http.bzl

```diff
@@ -502,6 +503,10 @@ unless it was added to the cache by a request with the same canonical id.
     "auth_patterns": attr.string_dict(
         doc = _AUTH_PATTERN_DOC,
     ),
+    "downloaded_file_path": attr.string(
```

Comment:
Can it just be a file name instead of a path? Or at least, we should make sure this can only be a relative path.

Reply:
Just a file name is probably best.